controllers/hibp.js

```diff
@@ -7,13 +7,21 @@ const HIBP = require("../hibp");
 
 
 async function notify (req, res) {
-  const hashes = req.body.hashSuffixes.map(suffix=>req.body.hashPrefix + suffix);
+  const reqBreachName = req.body.breachName.toLowerCase();
+  let breach = HIBP.getBreachByName(req.app.locals.breaches, reqBreachName);
+  if (!breach) {
+    // If breach isn't found, try to reload breaches from HIBP
+    await HIBP.loadBreachesIntoApp(req.app);
+    breach = HIBP.getBreachByName(req.app.locals.breaches, reqBreachName);
```

Comment:
Pretty sure we've discussed this before, but...

I'm not exactly sure how this works. AFAICT, logos are added manually, and we were experimenting w/ converting the SVGs to PNGs due to email compatibility issues. So if the breach isn't found in our current dataset and we sync w/ HIBP and then send out the email notifications for the new breach, we're going to be sending out emails w/ broken logo image for the latest breach.

/cc @groovecoder @lesleyjanenorton